src/setOfPeople.js

```diff
@@ -38,7 +38,15 @@
  * @return {Set}
  */
 function setOfPeople(people) {
-  // write code here
+  const playersSet = new Set();
```

Comment:
Don't use name of data types in variable name
```suggestion
  const players = new Set();
```